main.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const electron = require('electron');
+const ipc = electron.ipcMain;
+const app = electron.app;  // Module to control application life.
```

Comment:
I am not sure, this I took from electron's official Quick Guide. Can you double check which convention is to be used?